server/index.ts

```diff
@@ -19,9 +18,9 @@ const IS_PROD = MODE === 'production'
 const IS_DEV = MODE === 'development'
 const ALLOW_INDEXING = process.env.ALLOW_INDEXING !== 'false'
 
-const createRequestHandler = IS_PROD
-	? Sentry.wrapExpressCreateRequestHandler(_createRequestHandler)
-	: _createRequestHandler
+if (IS_PROD && process.env.SENTRY_DSN) {
+	import('../app/utils/monitoring.server.ts').then(({ init }) => init())
```

Comment:
This won't work when deployed because this file isn't bundled. Can we move the monitoring file to the server directory?

Reply:
Fixed!